src/AppBundle/Controller/NoteController.php

```diff
@@ -66,6 +66,25 @@ public function getNotesAction(Request $request, ParamFetcherInterface $paramFet
     }
 
     /**
+     * Presents the form to use to create a new note.
```

Comment:
If i'm not wrong this was already the case before :)
In fact we didn't change many things about the routing in 2.0.